src/conn/conn_compact.c

```diff
@@ -273,10 +292,12 @@ __background_compact_should_run(WT_SESSION_IMPL *session, const char *uri, int64
         compact_stat->skip_count++;
         conn->background_compact.files_skipped++;
         WT_STAT_CONN_INCR(session, background_compact_skipped);
```

Comment:
Can we move this stat to the caller and update it when skip is returned as true?

Reply:
If you looked the code, we update different stats in different situations. background_compact_exclude. Therefore, I still think we should do it in the function.

Reply:
All right but we still skip that file in the end.